### PR TITLE
[for 3.2.0] doc: prepare new release 3.2.0

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
Related to #3650 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] No test necessary here

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+
+### Fixed Issues
+
+- Frontend: [#3596](https://github.com/tracim/tracim/issues/3596)
+- Backend: [#3545](https://github.com/tracim/tracim/issues/3545),
+[#3660](https://github.com/tracim/tracim/issues/3660)
+
+## Known Issues
+
+- Its not possible to close sidebar by clicking around icon (issue [#3693](https://github.com/tracim/tracim/issues/3693))
```
Comment (gbonaspetti) [inline changelog.md:18]:
```suggestion
- It's not possible to close the sidebar by clicking around the icon (issue [#3693](https://github.com/tracim/tracim/issues/3693))
```

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,32 @@
...
+- Scroll bar on sidebar is not correctly integrated (issue [#3694](https://github.com/tracim/tracim/issues/3694))
+
+### Breaking/Important change
+
+- Renaming `Text document` app to `Note` (issue [#3621](https://github.com/tracim/tracim/issues/3621))
+- Renaming `Shared space` app to `Space` (issue [#3582](https://github.com/tracim/tracim/issues/3582))
```
Comment (gbonaspetti) [inline changelog.md:24]:
```suggestion
- Renaming `Shared space` to `Space` (issue [#3582](https://github.com/tracim/tracim/issues/3582))
```

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,32 @@
...
+- Webdav (issue [#3625](https://github.com/tracim/tracim/issues/3625)):
+  - It is no more possible to show two space with same name in webdav. If this case exist on Tracim, webdav show the first shared space id.
```
Comment (gbonaspetti) [inline changelog.md:26]:
```suggestion
  - It is no longer possible to show two spaces with the same name in webdav. If this case exist on Tracim, webdav show the first space id.
```

File: changelog.md [hunk 4]
```diff
@@ -1,3 +1,32 @@
...
+  - It is no more possible to create/modify space from webdav access
```
Comment (gbonaspetti) [inline changelog.md:27]:
```suggestion
  - It is no longer possible to create/modify a space from webdav access
```

**Comment by PhilippeAccorsi** [comment]
Wait to know if we can add this [issue](https://github.com/tracim/tracim/issues/3685) to this version. If yes, we need to add issue number in chnagelog.

File: changelog.md [hunk 5]
```diff
@@ -1,3 +1,32 @@
+
+## 3.2.0 / 2020-10-16
+
+### New Features
+
+- New spaces hierarchy and new sidebar view (a space can be child of another)
+- New access types for spaces (on request, open)
+- Automatic test can be done for Tracim docker image (feature for developers)
```
Comment (raphj) [inline changelog.md:8]:
```suggestion
- Spaces now can have sub-spaces  (a space can be child of another)
- The sidebar has been simplified and show the new space hierarchy
- New access types for spaces (on request, open)
- Automatic tests can be done for the Tracim docker image (feature for developers)
```

File: changelog.md [hunk 6]
```diff
@@ -1,3 +1,32 @@
...
+
+- It's not possible to close the sidebar by clicking around the icon (issue [#3693](https://github.com/tracim/tracim/issues/3693))
+- Scroll bar on sidebar is not correctly integrated (issue [#3694](https://github.com/tracim/tracim/issues/3694))
+
+### Breaking/Important change
+
+- Renaming `Text document` app to `Note` (issue [#3621](https://github.com/tracim/tracim/issues/3621))
+- Renaming `Shared space` to `Space` (issue [#3582](https://github.com/tracim/tracim/issues/3582))
+- Webdav (issue [#3625](https://github.com/tracim/tracim/issues/3625)):
+  - It is no longer possible to show two spaces with the same name in webdav. If this case exist on Tracim, webdav show the first space id.
```
Comment (raphj) [inline changelog.md:26]:
```suggestion
  - It is no longer possible to show two spaces with the same name in WebDAV. If two spaces at the same place have the same, will only show the first oldest space.
```

File: changelog.md [hunk 7]
```diff
@@ -1,3 +1,32 @@
...
+  - It is no longer possible to create/modify a space from webdav access
```
Comment (raphj) [inline changelog.md:27]:
```suggestion
  - It is no longer possible to create/modify a space from WebDAV
```

**Review by raphj — APPROVED** [review]

**Pull request merged**